app/views/scoreboards/index.html.erb

```diff
@@ -0,0 +1,15 @@
+<table>
+  <thead>
+    <tr>
+      <th>Name</th><th>Points</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% users.each do |user| %>
+      <tr>
+        <td><%= user.email.split('@').first.capitalize %></td>
```

Comment:
Please move it into a decorator.

https://github.com/drapergem/draper

Please also think about what exactly you want to display there:
``` s.lach@selleo.com'.split('@').first.capitalize ```  will return 'S.lach'.